React/Base/macOS/RCTUIKit.m

```diff
@@ -34,13 +34,14 @@ void UIGraphicsBeginImageContextWithOptions(CGSize size, __unused BOOL opaque, C
 		scale = 1.0;
 	}
 
-	size_t width = ceilf(size.width * scale);
-	size_t height = ceilf(size.height * scale);
+	size_t width = fmax(ceilf(size.width * scale), 1);
+	size_t height = fmax(ceilf(size.height * scale), 1);
```

Comment:
If we do go with this fix, we should briefly comment why it can't be zero

Reply:
Done